cdrc/fwv.go

```diff
@@ -178,6 +192,22 @@ func (self *FwvRecordsProcessor) recordToStoredCdr(record string, cdrcCfg *confi
 		duMultiplyFactor = cdrcCfg.DataUsageMultiplyFactor
 	}
 	for _, cdrFldCfg := range cfgFields {
+		// this part need to be added for fwv to filter a field from template
```

Comment:
any issue why you did not add it already? again, please len(filters) != 0 first.